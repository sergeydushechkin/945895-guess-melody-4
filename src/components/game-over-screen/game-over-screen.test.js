import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import GameOverScreen from "./game-over-screen.jsx";
import history from "../../history.js";

it(`Should GameOverScreen render correctly`, () => {
  const tree = renderer
    .create(
        <Router
          history={history}
        >
          <GameOverScreen
            onReplayButtonClick={() => {}}
          />
        </Router>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
