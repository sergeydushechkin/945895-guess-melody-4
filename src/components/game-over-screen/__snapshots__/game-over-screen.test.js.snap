// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GameOverScreen render correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Какая жалость!
  </h2>
  <p
    className="result__total result__total--fail"
  >
    У вас закончились все попытки. Ничего, повезёт в следующий раз!
  </p>
  <a
    className="replay"
    href="/"
    onClick={[Function]}
  >
    Попробовать ещё раз
  </a>
</section>
`;
